# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/patches/pkg_udev.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- udev-128/extras/scsi_id/scsi.h.orig	2008-08-26 09:21:40.000000000 +0200
+++ udev-128/extras/scsi_id/scsi.h	2008-09-17 16:22:22.000000000 +0200
@@ -70,9 +70,9 @@
 #define	SCSI_ID_ASCII	2
 
 struct scsi_id_search_values {
-	u_char	id_type;
-	u_char	naa_type;
-	u_char	code_set;
+	unsigned char	id_type;
+	unsigned char	naa_type;
+	unsigned char	code_set;
 };
 
 /*
--- udev-128/udev/udevd.c.orig	2008-09-10 21:37:29.000000000 +0200
+++ udev-128/udev/udevd.c	2008-09-17 16:04:48.000000000 +0200
@@ -47,6 +47,10 @@
 #include "udev.h"
 #include "udev_rules.h"
 
+#ifndef SO_RCVBUFFORCE
+#define SO_RCVBUFFORCE	(33)
+#endif
+
 #define UDEVD_PRIORITY			-4
 #define UDEV_PRIORITY			-2
 
